webview/src/setup/components/remotes/ShowExtension.tsx

```diff
@@ -2,14 +2,13 @@ import React from 'react'
 import styles from './styles.module.scss'
 import { Icon } from '../../../shared/components/Icon'
 import { Extensions } from '../../../shared/components/icons'
+import { ExtensionLink } from '../shared/ExtensionLink'
```

Comment:
[Q] Why not move the text out from the original paragraph, put it under the buttons and use this component? 

I think that would 

1. draw attention to the text
2. remove the need to show a tooltip.

Reply:
Good idea! Here's what that looks like so far:

<img width="678" alt="Screenshot 2023-06-19 at 11 01 48 AM" src="https://github.com/iterative/vscode-dvc/assets/43496356/68445011-1635-4637-bbd4-1233ce10823c">

I adjusted the paragraph styling so that it looks more connected to the disabled buttons vs a separate paragraph entirely.